src/common/transformations/src/transformations/common_optimizations/fuse_rotary_positional_embeddings.cpp

```diff
@@ -44,6 +44,7 @@
 
 using namespace ov::gen_pattern;
 using namespace ov::pass;
+using namespace ov::op;
```

Comment:
imho, using op::v1::... is more understandable )

Reply:
Some ops can have different versions
e.g v1 and v3